src/nni_manager/common/utils.ts

```diff
@@ -219,6 +219,11 @@ function getMsgDispatcherCommand(tuner: any, assessor: any, advisor: any, multiP
         if (advisor.classFileName !== undefined && advisor.classFileName.length > 1) {
             command += ` --advisor_class_filename ${advisor.classFileName}`;
         }
+        if (advisor.gpuIndices !== undefined) {
+            command = `CUDA_VISIBLE_DEVICES=${advisor.gpuIndices} ` + command;
+        } else {
```

Comment:
This env var setting is global for Tuner, Assessor and Advisor. It could conflict with tuner and assessor gpuNum settings.
Should we refine tuner, assessor advisor GPU settings together?